### DISCUSSION TITLE
DataTable gets taller than it should

### DISCUSSION THREAD

**Discussion by valerio-afk** [Textualize/textual#3481] [Q&A] [answered]
Hi all,

I have a big DataTable that may contains hundreds of rows.
My only issue so far, which I am not sure how to fix it, is that the last row is not visible, as it goes "underneath" the Footer of the app.
I know there is an extra row because I show some details in a Label in the app of each  highlighted row. If I go all down, I can see the details of this last row, although it below the Footer.

Is there a way to get the DataTable as tall as its container considering the Footer (or any docked Widgets?). 

The only style I have for the DataTable is `min-height: 5`, which I think it shouldn't have any implications to this issue.

Any suggestions on how to work around this issue is appreciated.

Cheers

**Comment by davep** [comment, marked as answer]
You will likely want to style your `DataTable` with a height that is appropriate for your application; see [the FAQ about `DataTable`](https://textual.textualize.io/FAQ/#why-doesnt-the-datatable-scroll-programmatically) for a little more context (I would suspect you would want to set the `height` to `1fr`, for example).

**Reply by valerio-afk** [reply]
Hi davep,

This seems have solved the issue. Thanks a lot!

**Reply by davep** [reply]
Glad to hear it solved it; most welcome. :-)
